app/src/main/java/ca/wheresthebus/MainDBViewModel.kt

```diff
@@ -163,15 +141,7 @@ class MainDBViewModel : ViewModel() {
         ).find().take(10)
```

Comment:
Unrelated to this PR, but if this is going to be a user setting, we should add it as a variable somewhere.

Reply:
👍